Tools/CISupport/ews-build/config.json

```diff
@@ -41,6 +41,7 @@
     { "name": "igalia18-wpe-ews", "platform": "wpe" },
     { "name": "igalia19-wpe-ews", "platform": "wpe" },
     { "name": "aperez-wpe-ews", "platform": "wpe" },
+    { "name": "playstation-ews-01", "platform": "playstation" },
```

Comment:
Do we want to add a set of them now @JonWBedard so all the password stuff can get sorted now?